docs/product-assembly_test-scenarios/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,37 @@
+[id='assembly_test-scenarios']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Testing a decision service using test scenarios
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: test-scenarios
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a business analyst or business rules developer, you can use test scenarios in {CENTRAL} to test a decision service before a project is deployed. Testing a decision service ensures that rule assets in the project function properly and as expected. You can test a decision service at any time during project development.
+
+.Prerequisites
+* The team and project for the decision service have been created in {CENTRAL}. For details, see {URL_GETTING_STARTED_DECISION_SERVICE}[_{GETTING_STARTED_DECISION_SERVICE}_].
+* Business rules and their associated data objects have been defined for the decision service. For example, see {URL_GUIDED_DECISION_TABLES}[_{GUIDED_DECISION_TABLES}_].
```

Comment:
Should this be For examples, see

Reply:
No, meaning that that guide is an example of way to define business rules (there are 5 ways - guided rules, decision tables, drl, etc.). I'll think on it for a clearer way.